bin/start

```diff
@@ -95,10 +98,19 @@ if [[ "$PROD" == "1" ]]; then
 else
     CMD="$CMD -f docker-compose.override.yaml"
 fi
+
 CMD="$CMD up"
+
 if [[ "$BUILD" == "1" ]]; then
     CMD="$CMD --build"
 fi
 
+if [[ "$LOG" == "1" ]]; then
```

Comment:
We want logs to be standard each time.  Is this leaving as an option instead of creating the logs every time?

Reply:
yup, we'll add the `-l` flag to our start command, like we do with the build `-b` flag